pkg/cmd/costmodel/costmodel.go

```diff
@@ -62,6 +61,13 @@ func StartExportWorker(ctx context.Context, model costmodel.AllocationModel) {
 		return
 	}
 	go func() {
+		// panic in a goroutine will crash the application. Log it and carry on.
+		defer func() {
+			if r := recover(); r != nil {
+				log.Errorf("panic in CSV Export Worker, stopping: %v", r)
+			}
+		}()
```

Comment:
I wonder if I should do another panic handler at the lower level (in addition to this one). So worker will keep trying to export according to the schedule (once a day).

But panic is something extraordinary that shouldn't happen.

Any preferences?

Reply:
I don't think a panic handler feels super necessary, agree panics should be rare. I am ok to to just skip the panic altogether actually